troubleshoot/ingest/fleet/common-problems.md

```diff
@@ -362,7 +362,7 @@ The ZIP archive containing diagnostics information will include the raw events o
 
 **Get the diagnostics bundle using the CLI**
 
-Run the following command to generate a zip archive containing diagnostics information that the Elastic team can use for debugging cases.
+Run the [`diagnostics` command](/reference/fleet/agent-command-reference#elastic-agent-diagnostics-command) to generate a zip archive containing diagnostics information that the Elastic team can use for debugging cases:
```

Comment:
My apologies, @stefnestor! I forgot the `.md` in the link

```suggestion
Run the [`diagnostics` command](/reference/fleet/agent-command-reference.md#elastic-agent-diagnostics-command) to generate a zip archive containing diagnostics information that the Elastic team can use for debugging cases:
```

Reply:
oh shiny, I like that that's simple 🎉